peachjam_search/serializers.py

```diff
@@ -72,8 +72,12 @@ def get_court(self, obj):
     def get_nature(self, obj):
         return obj["nature" + self.language_suffix]
 
-    def get_order_outcome(self, obj):
-        return obj["order_outcome" + self.language_suffix]
+    def get_order_outcomes(self, obj):
+        order_outcomes = obj["order_outcomes"]
+        if order_outcomes:
+            return [
+                order_outcome + self.language_suffix for order_outcome in order_outcomes
+            ]
```

Comment:
```suggestion
        return obj["order_outcomes" + self.language_suffix]
```

Reply:
@longhotsummer I'm getting this error when I apply your suggestion.
![image](https://github.com/laws-africa/peachjam/assets/15012985/d0461335-d70b-4cd2-b30b-9a9fdcd9bea2)